server/auth/auth.go

```diff
@@ -488,15 +583,39 @@ func (auth *AuthManager) Run(ctx context.Context) {
 	log.Infof("Allowing %d settling + taker order lots per market for new users.", auth.initTakerLotLimit)
 	log.Infof("Allowing up to %d settling + taker order lots per market for established users.", auth.absTakerLotLimit)
 
-	go auth.latencyQ.Run(ctx)
+	auth.wg.Add(1)
+	go func() {
+		defer auth.wg.Done()
+		t := time.NewTicker(20 * time.Second)
+		defer t.Stop()
+
+		for {
+			select {
+			case <-t.C:
+				auth.checkBonds()
+			case <-ctx.Done():
+				return
+			}
+		}
+	}()
```

Comment:
I wonder if it would be more efficient and accurate to prune the user's bonds at order time?

Reply:
Will look into pruning at order time, but issues I see there are (1) many orders in a narrow time period like an epoch and not checking too fast, and (2) we can't send tier change notifications closer to the time when their tier actually changes and they find out when trying to place an order.

Reply:
Also worth noting that bond check/prune involves no nodes requests, just a loop through the bond checking expiry times, so it's lightning fast.